Server/src/models/api.ts

```diff
@@ -40,7 +40,7 @@ export type TrackList = BareTrack[]
 /** @see {isCreatePOIType} ts-auto-guard:type-guard */
 export type CreatePOIType = {
 	name: string
-	icon: string
+	icon: number
```

Comment:
We should probably define the enum somewhere explicitly. Just `number` might be too broad. Ideally, we should try to make impossible things impossible to represent.